ts/ts-diabetes/src/main/java/se/inera/intyg/common/ts_diabetes/v3/rest/TsDiabetesModuleApiV3.java

```diff
@@ -211,4 +217,22 @@ public Certificate getCertificateFromJson(String certificateAsJson,
     public String getJsonFromCertificate(Certificate certificate, String certificateAsJson) throws ModuleException, IOException {
         throw new UnsupportedOperationException();
     }
+
+    @Override
+    public String getUtlatandeToInternalModelResponse(Utlatande utlatande) throws ModuleException {
+        if (utlatande instanceof TsDiabetesUtlatandeV3) {
+            return toInteralModelResponse((TsDiabetesUtlatandeV3) utlatande);
+        }
+        throw new IllegalArgumentException();
+    }
+
+    private String toInteralModelResponse(TsDiabetesUtlatandeV3 internalModel) throws ModuleException {
```

Comment:
No need to implement it again. An implementation already exists in the TsParentModuleApi

Reply:
Done.